stellargraph/data/explorer.py

```diff
@@ -352,6 +352,32 @@ def __init__(
         self.p = p
         self.q = q
         self.weighted = weighted
+        self._checked_weights = False
+
+        if weighted:
+            self._check_weights_valid()
+
+    def _check_weights_valid(self):
```

Comment:
The develop version also checks that multi-edges have the same weight, is this missing here? (or did I miss it??)

Reply:
Ah, yes, it's a behaviour change. This now properly handles a multigraph, previously it would pretend it was a non-multigraph, by just getting the weight of the first edge between the source/target nodes. Now it handles each edge independently, effectively summing the weights.

It should be highlighted.